internal/providers/terraform/azure/function_linux_app_test.go

```diff
@@ -0,0 +1,20 @@
+package azure_test
+
+import (
+	"testing"
+
+	"github.com/infracost/infracost/internal/providers/terraform/tftest"
+)
+
+func TestAzureRMLinuxAppFunctionGoldenFile(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+
+	opts := tftest.DefaultGoldenFileOptions()
+	opts.CaptureLogs = true
```

Comment:
Looks like you don't need to capture logs anymore and could add the t.Parallel back in if you want